lang/po/new/cs.po

```diff
@@ -46,14 +46,12 @@ msgid "[0;0] point offset"
 msgstr "[0;0] odsazeni bodu"
 
 # MSG_CRASH_DET_ONLY_IN_NORMAL c=20 r=4
-#: 
-msgid "Crash detection can\x0abe turned on only in\x0aNormal mode"
-msgstr "Crash detekce muze\x0abyt zapnuta pouze v\x0aNormal modu"
+msgid "Crash detection can\\x0abe turned on only in\\x0aNormal mode"

```

Comment:
Why are there `\\x0a` instead of `\x0a`? Did you change it or did POEdit change it automatically?

Reply:
POEdit change it.